index.html

```diff
@@ -9,11 +9,66 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
+
 <body>
+  <header>
+    <img class="img" src="./img/karma-logo.svg" alt="Logo"></img>
+    <nav class="logo1">
+      <ul class="list">
+        <li><a class="bar" href="index.html">Meet Karma</a></li>
+        <li><a href="#">How it Works</a></li>
+        <li><a href="store.html">Store</a></li>
+        <li><a href="#">Blog</a></li>
+        <li><a href="#">Help</a></li>
+        <li><a href="#">Login</a></li>
+      </ul>
+    </nav>
+  </header>
+  
+    <main>
+      <div class="karma">
+        <h1> Introducing karma <h1>
+        <p> Bring WiFi with you, everywhere you go.<p>
+        <button><a href="#">Learn More</a></button>  
```

Comment:
Using either a link or a button is good, you shouldn't combine them both together.

Remember that (generally...) a link is for directing the user to a new page, and a button is for an action.